my-cdk-app/lib/my-cdk-app-stack.ts

```diff
@@ -0,0 +1,67 @@
+import * as cdk from 'aws-cdk-lib';
+import * as s3 from 'aws-cdk-lib/aws-s3';
+import * as cloudfront from 'aws-cdk-lib/aws-cloudfront';
+import * as origins from 'aws-cdk-lib/aws-cloudfront-origins';
+import * as s3deploy from 'aws-cdk-lib/aws-s3-deployment';
+import { Construct } from 'constructs';
+
+export class MyCdkAppStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const websiteBucket = new s3.Bucket(this, 'XXXXXXXXXXXXX', {
```

Comment:
'XXXXXXXXXXXXX' name is quite interesting ) Maybe you will consider to adjust it.